default_weights.toml

```diff
@@ -0,0 +1,119 @@
+[base]
```

Comment:
Needs a comment at the top explaining what this file is for and that you can customize weights in the non-matching dir. Also needs a mention in the README.

Reply:
done